lib/middleware.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * Module dependencies.
+ * @private
+ */
+
+var kayvee = require("../lib/kayvee");
+var morgan = require("morgan");
+var _ = require("underscore");
+
+/**
+ * request path
+ */
+
+var getBaseUrl = function getBaseUrl(req) {
```

Comment:
I'm surprised this and the other function definitions in this file pass lint...usually it complains that this should be `function getBaseUrl(req) {`